tools/merge-shards/merge-shards.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+# The purpose of this script is a one time data migration from all of our 128
+# schemas into the public schema.
+
+set -euo pipefail
+
+usage() {
+  echo "Usage: merge-shards.sh <SERVER> [migrate]"
+  echo "Valid servers are: jobsrv, originsrv, sessionsrv"
+  exit 1
+}
+
+if [ -z "${PGPASSWORD:-}" ]; then
+  echo "Please ensure that PGPASSWORD is exported in your environment and run this script again."
+  exit 1
+fi
+
+echo "This script performs delicate surgery on your database."
+echo "It will take all of the data that's currently spread across 127 shards and consolidate all of that down into the public Postgres schema."
+echo "It performs some rudimentary error checking to ensure that all of the data was migrated successfully, but it's not exhaustive."
+echo "PLEASE ensure that you have completed a full database backup before running this script. If something goes horribly wrong, restoring from that backup is your only escape hatch."
```

Comment:
This verbiage is new?  I don't know if I missed this earlier.  The comments about delicate surgery and things going horribly wrong should to be reworded as that is probably not the type of text we should expose to end users.  We can suggest that users take a backup (and should then provide some instructions on how to do in the on-prem README separately).

Reply:
It's new.  I toned it down 🙂